pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/opplysningerbruktiberegningufoere/OpplysningerOmBarnetillegg.kt

```diff
@@ -120,66 +110,113 @@ data class OpplysningerOmBarnetillegg(
+                            fribeloepEllerInntektErPeriodisertSaerkullsbarn = saerkullTillegg.fribeloepEllerInntektErPeriodisert,
+                            harTilleggForFlereSaerkullsbarn = saerkullTillegg.harFlereBarn,
+                            harJusteringsbeloepSaerkullsbarn = harJusteringsbeloepSaerkullsbarn,
+                            sivilstand = sivilstand,
+                        )
+                    )
+                    includePhrase(JusteringBarnetillegg(saerkullTillegg.justeringsbeloepAar))
+                }
+            }.orShow {
+                //TBU605V (kun ett tillegg)
 
-            showIf(justeringsbeloepFelles.greaterThan(0)) {
-                includePhrase(JusteringBarnetillegg(justeringsbeloepFelles))
-            }.orShowIf(justeringsbeloepSaerkull.greaterThan(0)) {
-                includePhrase(JusteringBarnetillegg(justeringsbeloepSaerkull))
+                //Har kun felles tillegg
+                ifNotNull(barnetillegg.fellesbarn_safe) { fellesTillegg ->
+                    includePhrase(
+                        PeriodisertInntektInnledning(
+                            sivilstand = sivilstand,
+                            harKravaarsakEndringInntekt = harKravaarsakEndringInntekt,
+                            harFlereTillegg = false.expr(),
+                        )
+                    )
+
+                    showIf(fellesTillegg.beloepNetto.notEqualTo(fellesTillegg.beloepBrutto)) {
+                        includePhrase(
+                            PeriodisertInntektOverFribeloepEttTillegg(
+                                inntektEllerFribeloepErPeriodisert = fellesTillegg.fribeloepEllerInntektErPeriodisert,
+                                avkortningsbeloepAar = fellesTillegg.avkortningsbeloepAar,
+                                harJusteringsbeloep = harJusteringsbeloepFellesBarn,
+                            )
+                        )
+                    }
+
+                    showIf(harJusteringsbeloepFellesBarn) {
+                        includePhrase(JusteringBarnetillegg(justeringsbeloepFelles))
+                    }
+                }
+
+                //Har kun særkull tillegg
+                ifNotNull(barnetillegg.saerkullsbarn_safe) { saerkullsTillegg ->
+                    includePhrase(
+                        PeriodisertInntektInnledning(
+                            sivilstand = sivilstand,
+                            harKravaarsakEndringInntekt = harKravaarsakEndringInntekt,
+                            harFlereTillegg = false.expr(),
+                        )
+                    )
+
+                    showIf(saerkullsTillegg.beloepNetto.notEqualTo(saerkullsTillegg.beloepBrutto)) {
+                        includePhrase(
+                            PeriodisertInntektOverFribeloepEttTillegg(
+                                inntektEllerFribeloepErPeriodisert = inntektEllerFribeloepErPeriodisert,
```

Comment:
Hvorfor  er er denne annerledes enn for tilsvarende for fellesbarn?

Reply:
`fellesTillegg.fribeloepEllerInntektErPeriodisert`vs `inntektEllerFribeloepErPeriodisert`